unstructured/ingest/evaluate.py

```diff
@@ -39,14 +14,14 @@ def main():
 
 @main.command()
 @click.option("--output_dir", type=str, help="Directory to structured output.")
+@click.option("--source_dir", type=str, help="Directory to source.")
```

Comment:
If this reordering works with the `generate_args` function in `evaluation-metrics.sh` line 41 then it works for me too.

Reply:
Yep! The args can still be formed and passed in any order from this script because they're named arguments